examples/p5js/ObjectDetector/ObjectDetector_COCOSSD_Video/sketch.js

```diff
@@ -37,8 +40,8 @@ function modelReady() {
 function draw() {
   image(video, 0, 0);
 
-  for (let i = 0; i < detections.length; i++) {
-    let object = detections[i];
+  for (let i = 0; i < detections.length; i += 1) {
```

Comment:
I think our linting is a bit overly strict with some of these requirements (for the examples) but this is low priority and we can revisit later!